packages/ai/mcp-stdio/create-child-process.ts

```diff
@@ -1,6 +1,6 @@
 import type { ChildProcess } from 'node:child_process';
-import { MCPClientError } from '../../../errors';
-import { McpStdioServerConfig } from './types';
+import { MCPClientError } from '../errors';
+import { McpStdioServerConfig } from './mcp-stdio-transport';
```

Comment:
we might be able to reverse some of the hacks in here (no dynamic imports needed if we do separate bundling, so the hack become accidental complexity that should be removed)